com.unity.render-pipelines.high-definition/Runtime/Sky/AmbientProbeConvolution.compute

```diff
@@ -12,12 +12,12 @@ TEXTURECUBE(_AmbientProbeInputCubemap);
 
 // Constants from SetSHEMapConstants function in the Stupid Spherical Harmonics Tricks paper:
 // http://www.ppsloan.org/publications/StupidSH36.pdf
-#define sqrtPI (sqrt(PI))
-#define fC0 (0.282095 * (1.0 / (2.0 * sqrtPI)))
-#define fC1 (0.488603 * (sqrt ( 3.0) / ( 3.0 * sqrtPI)))
-#define fC2 (1.092548 * (sqrt (15.0) / ( 8.0 * sqrtPI)))
-#define fC3 (0.315392 * (sqrt ( 5.0) / (16.0 * sqrtPI)))
-#define fC4 (0.546274 * 0.5 * fC2)
+//  [SH basis coeff] * [clamped cosine convolution factor]
+#define fC0 (rsqrt(PI * 4.0) * rsqrt(PI * 4.0)) // Equivalent to (0.488603 * (sqrt ( 3.0) / ( 3.0 * sqrtPI)))
+#define fC1 (rsqrt(PI * 4.0 / 3.0) * rsqrt(PI * 3.0)) // Equivalent to (1.092548 * (sqrt (15.0) / ( 8.0 * sqrtPI)))
+#define fC2 (rsqrt(PI * 4.0 / 15.0) * rsqrt(PI * 64.0 / 15.0)) // Equivalent to (1.092548 * (sqrt (15.0) / ( 8.0 * sqrtPI)))
+#define fC3 (rsqrt(PI * 16.0 / 5.0) * rsqrt(PI * 256.0 / 5.0)) // Equivalent to (0.315392 * (sqrt ( 5.0) / (16.0 * sqrtPI)))
```

Comment:
here is off too 

In general I added a desmos comments vs factors  https://www.desmos.com/calculator/xrn75bj2gz

Reply:
for fc3 you missed a PI in desmos, now all match

Reply:
Ah right! My bad :D

Reply:
Was reported by a user in March but have forget and just rethinked about it :)